src/widget/woverview.cpp

```diff
@@ -1005,9 +1016,18 @@ void WOverview::drawMarks(QPainter* pPainter, const float offset, const float ga
         pPainter->drawLine(line);
 
         if (rect.isValid()) {
-            QColor loopColor = pMark->fillColor();
-            loopColor.setAlphaF(0.5f);
-            pPainter->fillRect(rect, loopColor);
+            QColor rangeColor = pMark->fillColor();
+            // Less opacity for inactive jump cues to not unnecessarily obstruct
+            // the waveform image.
+            // TODO Use played color for forward jumps to clarify we'll skip that region?
+            if (pMark->getType() == mixxx::CueType::Jump && pMark->isActive()) {
+                rangeColor.setAlphaF(0.2f);
+            } else {
+                rangeColor.setAlphaF(0.5f);
```

Comment:
Whoopsy.. please swap the values
```suggestion
                rangeColor.setAlphaF(0.5f);
            } else {
                rangeColor.setAlphaF(0.2f);
```